odpc.client/src/components/TheHeader.vue

```diff
@@ -11,6 +11,10 @@
         <li v-if="user?.isAdmin">
           <router-link :to="{ name: 'gebruikersgroepen' }">Gebruikersgroepen</router-link>
         </li>
+
+        <li>
+          <a href="/api/logoff" title="Uitloggen">Uitloggen</a>
```

Comment:
is uitloggen ooit gespect? is wel duidelijk wat er dan moet gebeuren en gebeurt dat ook?

Reply:
goed punt, is nog niet gespect. we schuiven het op de lange baan.